### ISSUE TITLE
The results could not be replicated.

### ISSUE THREAD

**Issue by TimeLearner1** [TROUBADOUR000/TimeFilter#12]
Thank you for open-sourcing your code, but I was unable to reproduce the results using your provided hyperparameters. Furthermore, there is a bug in your paper: you identify the best result even when the test set loss is increasing. This bug has been mentioned in previous comments but remains unfixed. There are significant differences in the results, particularly for ETTh1, 96 predicting 720, where training stopped after only one epoch. 

**Your reported results are 0.448 and 0.457,** 

**The reproduced results are mse: 0.511, mae: 0.485.**
